common/datavalues/src/types/data_type.rs

```diff
@@ -273,6 +289,12 @@ impl From<&ArrowDataType> for DataType {
                 DataType::Struct(fields)
             }
 
+            ArrowDataType::Dictionary(_, val) => match val.as_ref() {
+                ArrowDataType::UInt8 => DataType::UInt8,
```

Comment:
@sundy-li I am not quite clear if this is correct. I thought it cannot convert to 
```rust
Enum8(BtreeMap<String,u8>)
```